common/big.go

```diff
@@ -26,6 +26,7 @@ var (
 	Big0     = big.NewInt(0)
 	BigTrue  = Big1
 	BigFalse = Big0
+	Big10    = big.NewInt(10)
```

Comment:
Please remove this one, it's only being used once.

And perhaps this is more a general notion I don't want to use these silly `common` obscure and dangerous variables anymore. Put them in the package they are used instead (even when dup).